cmd/common.go

```diff
@@ -0,0 +1,14 @@
+package cmd
+
+import "os"
+
+func resolveEnvironment(args []string, defaultEnvironment string) string {
```

Comment:
At quite short term, I think this will not be the right place for this function as it may be required by the config, app, ... Probably it will have to be in a "common" package or a "utils" (if not dependant from "ALGOREA"). 
But anyway, the move may be done later.